test/provision/manifest/dns_deployment.yaml

```diff
@@ -1,30 +1,15 @@
 ---
 apiVersion: v1
-kind: ConfigMap
-metadata:
-  name: kube-dns
-  namespace: kube-system
-  labels:
-    addonmanager.kubernetes.io/mode: EnsureExists
----
-apiVersion: v1
```

Comment:
Any particular reason why we are removing the ServiceAccount?

Reply:
At the moment is created two times: 

First: 
https://github.com/cilium/cilium/blob/8b1d22d072890341fffb3919200876578a264629/test/provision/manifest/dns_deployment.yaml#L10-L17

Second: 
https://github.com/cilium/cilium/blob/8b1d22d072890341fffb3919200876578a264629/test/provision/manifest/dns_deployment.yaml#L41-L48